pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -68,7 +68,6 @@ stdenv.mkDerivation rec {
     cd "$out/etc/fonts"
     rm conf.d/{50-user,51-local}.conf
```

Comment:
I don't think that commit is important. I just didn't want to invest enough time for a truly proper solution, like this PR attempts (I suppose). (At that time it's possible I didn't even know that fontconfig doesn't interleave the directories.)